airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_manifest_only/manifest_resolver.py

```diff
@@ -137,15 +137,15 @@ def _lookup_ref_value(self, ref: str, manifest: Mapping[str, Any]) -> Any:
             raise ValueError(f"{path}, {ref}")
 
     @staticmethod
-    def _is_ref(node: Any) -> bool:
+    def _is_ref(node: Any) -> bool:  # noqa: ANN401
         return isinstance(node, str) and node.startswith("#/")
 
     @staticmethod
     def _is_ref_key(key: str) -> bool:
         return bool(key == REF_TAG)
 
     @staticmethod
-    def _read_ref_value(ref: str, manifest_node: Mapping[str, Any]) -> Any:
+    def _read_ref_value(ref: str, manifest_node: Mapping[str, Any]) -> Any:  # noqa: ANN401
```

Comment:
Should we change the linter rules instead?

Reply:
No, we should more accurately type this module 😄